src/scripting/Scripting.cpp

```diff
@@ -135,8 +135,7 @@ void Scripting::PostInitializeStage1()
 
     luaVm.new_usertype<Scripting>("__Game",
         sol::meta_function::construct, sol::no_constructor,
-        sol::meta_function::index, &Scripting::Index,
-        sol::meta_function::new_index, &Scripting::NewIndex);
```

Comment:
If new_index is removed, you can just remove InternalNewIndex?

Reply:
Yeah, we can remove it.

Reply:
And InternalIndex too actually, it's used in Index only.